inbm/dispatcher-agent/dispatcher/aota/application_command.py

```diff
@@ -151,6 +183,8 @@ def update(self) -> None:
 
         logger.debug(f"driver path = {driver_path}")
         try:
+            self.run_signature_check(driver_path)
+            
             if not self._is_rpm_file_type(driver_path):
                 raise AotaError('Invalid file type')
```

Comment:
**Suggestion:** Use a context manager when working with file paths to ensure that resources are properly managed and to handle exceptions more gracefully. [best practice]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
            with contextlib.suppress(FileNotFoundError):
                for file in os.listdir(CENTOS_DRIVER_PATH):
                    remove_file(os.path.join(CENTOS_DRIVER_PATH, file))
```